proposed/extended-coding-style-guide-meta.md

```diff
@@ -0,0 +1,96 @@
+Extended Coding Style Guide Meta Document
+=========================================
+
+1. Summary
+----------
+
+This document describes the process and discussions that led to the Extended Coding
+Style PSR. Its goal is to explain the reasons behind each decision.
+
+2. Why Bother?
+--------------
+
+PSR-2 was accepted in 2012 and since then a number of changes have been made to PHP,
+most notably recent changes for PHP 7, which have implications for coding style
+guidelines. Whilst PSR-2 is very comprehensive of PHP functionality that existed at
+the time of writing, new functionality is very open to interpretation. PSR-N seeks
+to provide a set way that both coding style tools can implement, projects can declare
+adhearance to and developers can easily relate on between different projects for these
```

Comment:
Yeah I wasn't sure about the relate bit. I think this whole clause could be reworded potentially.